client/tunnel

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+if [[ $# -eq 0 ]] ; then
+    echo "Please provide the hostname:port as argument, for example $0 host_inside_vpc:3306"
+    exit 0
+fi
+
+
+REGION=eu-central-1
+NAME=bastion-ssm-iam
+DEV_LOCAL_PORT=$(( ( RANDOM % 60000 ) + 1024 ))
+FORWARD_TO=$1
+INSTANCE_ID=$(aws ec2 describe-instances \
+               --region $REGION \
+               --filter "Name=tag:Name,Values=$NAME" \
+               --query "Reservations[].Instances[?State.Name == 'running'].InstanceId[]" \
+               --output text)
+
+DESCRIBE_EC2_JSON=$(aws ec2 describe-instances --region $REGION --filter "Name=tag:Name,Values=$NAME" --output json)
+
+AZ=$(echo $DESCRIBE_EC2_JSON |jq -r '.Reservations[].Instances[].Placement.AvailabilityZone')
+INSTANCE_ID=$(echo $DESCRIBE_EC2_JSON |jq -r '.Reservations[].Instances[].InstanceId')
+
+aws ec2-instance-connect send-ssh-public-key --region $REGION --instance-id $INSTANCE_ID --availability-zone $AZ --instance-os-user ec2-user --ssh-public-key file://$HOME/.ssh/id_rsa.pub
```

Comment:
I'm not using `$HOME/.ssh/id_rsa.pub`... So I would need to pass this as environment variable. `$HOME/.ssh/id_rsa.pub` could be default.

Reply:
I fixed that to also allow `id_ed25519.pub`, for everything else people can hack the script.